test/xml/generated/Xml/models/xmlMappers.ts

```diff
@@ -20,6 +20,8 @@ export {
   ContainerProperties,
   CorsRule,
   ErrorModel,
+  JSONInput,
```

Comment:
Do you have any idea why all of the `JSONInput` and `JSONOutput` code is now being generated?  I don't see a relevant spec change.  Perhaps this is coming from a standard Swagger spec file from another repo?

Reply:
Yes. It is from autorest.testserver

Reply:
Thanks!  Looks like it might have come from this PR: https://github.com/Azure/autorest.testserver/pull/87.  PR looks good, ship it!